recipes/mizuroute/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export F_MASTER="../"
+export FC_EXE=${FC}
+export EXE="route_runoff.exe"
+export MODE="fast"
+export NCDF_PATH=${PREFIX}
+export EXE_PATH="${PREFIX}/bin"
+export FLAGS="-O3 -fmax-errors=0 -ffree-line-length-none"
+
+patch route/build/Makefile ${RECIPE_DIR}/make.patch
```

Comment:
You should probably put this in the meta.yaml: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Reply:
I think it's fine the way it is. As before, this is just going to get removed in a future release.

Reply:
Well it is better for conda to know about the patch